### PR TITLE
clarify about same ECN code point, add formula for smallest and large…

### PR DESCRIPTION
**Pull request by goelvidhi**
I attempted to clarify by adding some more details and an example. Probably this PR is going to need more updates. Looking forward to hear your feedback.

### CODE REVIEW

File: draft-seemann-quic-accurate-ack-ecn.md [hunk 1]
```diff
@@ -86,6 +88,10 @@ ACCURATE_ACK_ECN Frame {
 Except for the two ACK Range fields, all the fields are the same as defined in
 {{Section 19.3 of RFC9000}}.
 
+All packets within an ACK range MUST be received with the same ECN code point.
```
Comment (marten-seemann) [inline draft-seemann-quic-accurate-ack-ecn.md]:
```suggestion
All packets within an ACK range MUST have been received with the same ECN code point.
```

File: draft-seemann-quic-accurate-ack-ecn.md [hunk 2]
```diff
@@ -35,28 +35,30 @@ informative:
...
-QUIC defines a variant of the ACK frame that carries three counters, one for the
-ECT(0), one for the ECT(1) and one for the CE codepoint. From this information,
+QUIC defines a variant of the ACK frame that carries cumulative count for
+each of the three ECN codepoints (ECT(1), ECT(0) and CE). From this information,
 the recipient of the ACK frame cannot deduce which ECN marking the individual
 packets were received with.
 
 This document defines an alternative ACK frame that encodes enough information
-to indicate which ECN marks each individual packet was received with..
+to indicate which ECN mark each individual packet was received with at the receiver.
```
Comment (marten-seemann) [inline draft-seemann-quic-accurate-ack-ecn.md]: "received with at the receiver" sounds weird.

```suggestion
to indicate which ECN mark each individual packet was received with.
```

**Review by marten-seemann — APPROVED** [review]
Thank you @goelvidhi, and sorry for the long time it took me to review the PR!

**Comment by goelvidhi** [comment]
@marten-seemann are there any more comments or is this ready to merge?

**Review by marten-seemann — APPROVED** [review]

**Comment by marten-seemann** [comment]
Thank you for this PR @goelvidhi, this was very helpful!

**Pull request merged**